python/lsst/jointcal/jointcal.py

```diff
@@ -242,37 +257,39 @@ def run(self, ref, tract):
         # Build an updated wcs for each calexp
         imList = assoc.TheCcdImageList()
 
-        for im in imList :
+        for im in imList:
             tanSip = spm.ProduceSipWcs(im)
             frame = im.ImageFrame()
             tanWcs = afwImage.TanWcs.cast(jointcalLib.GtransfoToTanWcs(tanSip, frame, False))
 
             name = im.Name()
             visit, ccd = name.split('_')
-            for dataRef in ref :
-                if dataRef.dataId["visit"] == int(visit) and dataRef.dataId["ccd"] == int(ccd) :
+            for dataRef in dataRefs:
+                if dataRef.dataId["visit"] == int(visit) and dataRef.dataId["ccd"] == int(ccd):
```

Comment:
I know this is existing code, but is this safe? I don't think DECam uses "ccd" (it uses "ccdnum" instead). Similarly, is it clear that the name can be split like that? It all seems quite specific to one camera. I don't know if fixing that is in scope for this ticket, but if not, please add it to a different existing ticket or file a new one.